lib/src/interfaces.dart

```diff
@@ -25,4 +21,12 @@ abstract class AdapterInterface {
   AdapterRequest get onPut;
   AdapterRequest get onDelete;
   AdapterRequest get onPatch;
+  dynamic throwError(Responsable response);
+}
+
+/// Top level interface for [Dio]'s [ResponseBody] amd also [Dio]'s [DioError].
```

Comment:
[ResponseBody] amd also [Dio]'s -> [ResponseBody] and also [Dio]'s

Reply:
???

Reply:
I think @nataliehippe means that `amd` should be `and`. A typo in short.